examples/fun3d_examples/_ssw-inviscid/_test_1_derivs.py

```diff
@@ -243,12 +243,12 @@
     start_time = time.time()
 
     # run the finite difference test
-    max_rel_error = TestResult.derivative_test(
+    max_rel_error = TestResult.finite_difference(
```

Comment:
Can you add the AitkenRelaxationTacs object to this example so we know how to use it in the Tacs.create_from_bdf?